README.md

```diff
@@ -24,6 +24,26 @@ env_vars:
 
 This role will update variables' value if necessary.
 
+## Pam environment variables - .pam_environment
+
+In case you want to narrow down visibility of your environment variables to only one user (for example: www-data) and not 
+expose them to everyone use env_pam_* options to set it. Read more: https://help.ubuntu.com/community/EnvironmentVariables#A.2BAH4-.2F.pam_environment
+
+```
+env_pam_path: "/var/www" # check $HOME directory for www-data user
+env_pam_module_arguments: "readenv=1 user_readenv=1" 
+env_pam_owner: "www-data"
+env_pam_vars: {
+    - { regexp: '^TEST=', line: 'TEST=new_test_variable' }
```

Comment:
this way of setting pam env vars is very limiting, you cant set env vars for 2 different users.

Reply:
Config has been updated to have only one env_pam_users variable